author.tf

```diff
@@ -310,19 +310,56 @@ module "author-api" {
   auth_unauth_action         = "deny"
 
   container_environment_variables = <<EOF
-      {
-        "name": "DB_CONNECTION_URI",
-        "value": "postgres://${var.author_database_user}:${var.author_database_password}@${module.author-database.database_address}:${module.author-database.database_port}/${var.author_database_name}"
-      },
       {
         "name": "RUNNER_SESSION_URL",
         "value": "${module.author-survey-runner.service_address}/session?token="
       },
       {
         "name": "PUBLISHER_URL",
         "value": "${module.publisher.service_address}/publish/"
+      },
+      {
+        "name": "DYNAMO_QUESTIONNAIRE_TABLE_NAME",
+        "value": "${module.author-dynamodb.author_questionnaires_table_name}"
+      },
+      {
+        "name": "DYNAMO_QUESTIONNAIRE_VERSION_TABLE_NAME",
+        "value": "${module.author-dynamodb.author_questionnaire_versions_table_name}"
       }
   EOF
+
+  task_has_iam_policy = true
+
+  task_iam_policy_json = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+      {
+          "Sid": "",
+          "Effect": "Allow",
+          "Action": [
+              "dynamodb:Scan",
+              "dynamodb:DescribeTable",
+              "dynamodb:PutItem",
+              "dynamodb:GetItem",
+              "dynamodb:DeleteItem"
+          ],
+          "Resource": "${module.author-dynamodb.author_questionnaires_table_arn}"
+      },
+      {
+          "Sid": "",
+          "Effect": "Allow",
+          "Action": [
+              "dynamodb:DescribeTable",
```

Comment:
what is `dynamodb:DescribeTable` used for?

Reply:
Dynamoose required this permission when defining the models.